src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.NoSuchElementException;
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDaoImpl) {
+        actorDao = actorDaoImpl;
+    }
```

Comment:
write constructor in this way. This is common and good practice 
```suggestion
    public ActorServiceImpl(ActorDao actorDao) {
        this.actorDao = actorDao;
    }
```